hardhat/tasks/sips/args/sipArgs.js

```diff
@@ -1249,6 +1249,91 @@ const getArgsSip0087 = async (hre) => {
     return { args, governor: "GovernorOwner" };
 };
 
+const getArgsSip0088 = async (hre) => {
```

Comment:
pls remove rollover and liquidations cuz they aren't updated

Reply:
updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/561/commits/0b43a2ffcce40d7410e1d9f72d92581a5e92c220)